server/src/controllers/v1/cards/completion.js

```diff
@@ -0,0 +1,19 @@
+const { Card } = require('../../../models')
```

Comment:
@datarockets/cuties thoughts about going with `import` if possible for consistence?

`import { Card } from '../../../models'`
`const { Card } = require('../../../models')`

Reply:
@AleksSenkou Blet, linter again 😔. When we use import it is works, but linter write 
<img width="715" alt="2018-12-20 16 48 17" src="https://user-images.githubusercontent.com/32716983/50288523-364ac580-0477-11e9-82a1-ec9d51e81665.png">

Reply:
@nikitasakov have you tried to fix it?

Reply:
@AleksSenkou Yes, I tried, but it looks more like a bug of eslint, as I understand eslint just wants explicit export of `Card` in `models/index`, otherwise it just does not see `Card`. But we cannot make an explicit export of all models.

Reply:
@nikitasakov I see. Can we go with infra task for now?